packages/wallet-sdk/src/relay/aes256gcm.test.ts

```diff
@@ -12,11 +12,12 @@ describe("aes256gcm", () => {
     expect(encrypted.length).toEqual(90);
 
     // decrypted output matches original input
-    decrypt(encrypted, randSecret).subscribe({
-      next: decrypted => {
+    return decrypt(encrypted, randSecret).then(
+      decrypted => {
         expect(decrypted).toBe("plain text string");
       },
-    });
+      () => {},
+    );
```

Comment:
nit: can we use async/await in tests? A bit easier to read (same below)

Reply:
yup sure.